### ISSUE TITLE
CI: update matrix

### ISSUE THREAD

**Issue by blueyed** [pytest-dev/pytest-cov#239]
I suggest only testing pytest 3 (`<4.0`) and above, i.e `t310` and `t40`.
Also having pytest-master in the matrix would be helpful.

What do you think?

**Comment by blueyed** [comment]
Started at https://github.com/pytest-dev/pytest-cov/pull/240.
The current matrix is quite ridiculous big IMHO.

**Comment by ionelmc** [comment]
I agree about supporting only 2 pytest versions. We could even only test with the latest - people that need old pytest can simply use older pytest-cov (it's not like past pytest-cov releases are horrendously broken).
